### PR TITLE
fix added warning IP list too large

### PR DESCRIPTION
**Pull request by franciscojoseabellan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by MosheEichler — COMMENTED** [review]
Hi @franciscojoseabellan,
Thank you for your contribution!
Good work :)

Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by franciscojoseabellan** [comment]
Hello [MosheEichler](https://github.com/MosheEichler)

After executing the command, we got notified that no rellevant changes were made in this commit

image

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py [hunk 1]
```diff
@@ -1337,6 +1337,7 @@
             "use_ip_nt_range_tags_include", "use_ip_nt_range_tags_exclude",
             "active",
             "scanners_in_network",
+            "fqdn",
```
Comment (MosheEichler) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.py:1340]: why did you add it

**Comment by MosheEichler** [comment]
> Hello [MosheEichler](https://github.com/MosheEichler)
> 
> After executing the command, we got notified that no rellevant changes were made in this commit
> 
> image

Hey @franciscojoseabellan
Don't wory I'll take care on it.
Please see my comment.

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py (hunk 1 above)
Comment (franciscojoseabellan) [reply Packs/qualys/Integrations/Qualysv2/Qualysv2.py:1340]: I added it because Qualysv2 lacked the parameter fqdn
If not added, Demisto returns an error:

scan_create_no_fqdn

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.py [hunk 2]
```diff
@@ -1488,7 +1489,10 @@ def __init__(self, base_url, username, password, verify=True, proxy=False, heade
 
     @staticmethod
     def error_handler(res):
-        err_msg = f"Error in API call [{res.status_code}] - {res.reason}"
+        err_msg = ""
+        if res.status_code == 414 or res.status_code == 520:
```
[the remark this answers is not in the document]
Comment (franciscojoseabellan) [reply Packs/qualys/Integrations/Qualysv2/Qualysv2.py:1493]: It is an error when we provide to the ip field an string of approximately between 5000 and 12000 characters containing IP, we receive a 414 error from the API, when we provide more than approximately 12000 characters containing IPs, we receive a 520 error,

We do not know if it is an error related to the Demisto processing of large characters in the URL

image

image

**Comment by MosheEichler** [comment]
Hi @franciscojoseabellan, the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Review by MosheEichler — APPROVED** [review]

**Comment by MosheEichler** [comment]
> [CLA assistant check](https://cla-assistant.io/demisto/content?pullRequest=30217) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=30217) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=30217) it.

Hey @franciscojoseabellan
Please sing the CLA

**Comment by franciscojoseabellan** [comment]
Hi @MosheEichler,

Can I find you with the same username (MosheEichler) on Slack to talk about the demo?

Regards,

**Comment by MosheEichler** [comment]
> Hi @MosheEichler,
> 
> Can I find you with the same username (MosheEichler) on Slack to talk about the demo?
> 
> Regards,

Yes. search Moshe Eichler.

**Comment by franciscojoseabellan** [comment]
@MosheEichler 

Do you know when this PR would be incorporated in the master branch of demisto?

Thanks for your help

Regards,
